src/Database/LSMTree/Internal/RunAcc.hs

```diff
@@ -265,7 +264,6 @@ addLargeSerialisedKeyOp racc@RunAcc{..} k page overflowPages =
   assert (RawPage.rawPageOverflowPages page > 0) $
   assert (RawPage.rawPageOverflowPages page == length overflowPages) $ do
     modifyPrimVar entryCount (+1)
-    Bloom.insert mbloom k
```

Comment:
This one should probably remain, because large entries bypass the page accumulator, and therefore `flushPageIfNonEmpty` is bypassed, and `bloomInserts` is bypassed